NtupleTools/python/templates/trigger.py

```diff
@@ -54,10 +54,6 @@
         name='singleE_leg2', 
         paths=r'HLT_Ele12_CaloIdL_TrackIdL_IsoVL_v\\d+'
         ),
-    _trig_template.replace(
```

Comment:
I didn't remove it. I had added it to my release before somebody else added it to the main branch list at a different position. So when  pulled from the main release it got added twice. I just removed one of the double additions. It's still there.